### ISSUE TITLE
Labeled list indentation in 6.8+ darkfish is inconsistent with previous versions

### ISSUE THREAD

**Issue by flavorjones** [ruby/rdoc#1227]
Labeled list indentation seems to have changed, and I'm not sure that it was an intentional change.

Previously, the list syntax `[title]` always had a visual line break before the value, while `title::` stayed on the same line. Now, the value always starts on the same line as the title.

From the RDoc::MarkupReference docs, here's how it renders in older darkfish:

![image](https://github.com/user-attachments/assets/6498a51e-209a-4a8b-b780-432668fc3ffe)

And here's how it renders with the style changes in 6.8:

![image](https://github.com/user-attachments/assets/de9924f1-6460-4796-a3d7-69c69fbd9b57)

As a result, markup like this:

```
# [Parameters]
# - +input+ (String) RELAX NG schema definition
# - +options:+ (Nokogiri::XML::ParseOptions)
#   Defaults to ParseOptions::DEFAULT_SCHEMA
```

which used to render well:

![image](https://github.com/user-attachments/assets/08280fb9-a77b-418f-9413-28dc16922492)

now renders oddly:

![image](https://github.com/user-attachments/assets/68b891ca-6dbe-434e-8c3b-e1eb75e8ea51)

I'll take a look in the next few days, but wanted to capture what I'm seeing before I forget.

**Comment by st0012** [comment]
Seems to be related to #1199 too

**Comment by flavorjones** [comment]
@st0012 Yes, I hadn't seen that one, but they are related.

Here's a more complex example of the strange styling with labeled lists:

```
      # [Required Parameters]
      # - +input+ (String | IO): The \XML document to parse.
      #
      # [Optional Parameters]
      # - +url:+ (String) The base URL of the document.
      # - +encoding:+ (String) The name of the encoding of the document.
      # - +options:+ (Integer | ParseOptions) Options to control the parser behavior.
      #   Defaults to +ParseOptions::STRICT+.
      #
      # [Yields]
      # If present, the block will be passed a Nokogiri::XML::ParseOptions object to modify before
      # the fragment is parsed. See Nokogiri::XML::ParseOptions for more information.
```

![image](https://github.com/user-attachments/assets/d904d7fa-7336-401b-b6b6-025cc73b9f7d)

**Comment by flavorjones** [comment]
#1209 is a great fix for this issue, I +1ed it there.

**Comment by st0012** [comment]
Closing as I've merged #1209
